pkg/controller/session/session_controller_test.go

```diff
@@ -40,8 +42,19 @@ var _ = Describe("Basic session manipulation", func() {
 			return s
 		}
 	}(&c)
+	GetStatusRef := func(name string, session v1alpha1.Session) v1alpha1.RefStatus {
+		for _, ref := range session.Status.Refs {
+			if ref.Name == name {
+				return *ref
+			}
+		}
+		return v1alpha1.RefStatus{}
```

Comment:
Is it supposed to mean "not found"?

Reply:
https://github.com/Maistra/istio-workspace/pull/279/commits/dcb86fa04166e3ba6dd88fc419b71138fa7f4d22